storage_service/administration/views.py

```diff
@@ -25,6 +26,29 @@ def settings_edit(request):
     return render(request, 'administration/settings_form.html', locals())
 
 
+########################## VERSION ########################
+
+def get_ss_version():
+    try:
+        from storage_service import __version__ as ss_version
```

Comment:
(I assume the ImportError comment below was supposed to be here).  I'd remove the try/catch here - we want the ImportError to bubble up, since if that variable has disappeared that's a bug we don't want to silence.  Also, since it's within our codebase, it seem safe to assume that it will be there (or any change removing it will fix anything that depends on it).